programStructure/Chessboard.js

```diff
@@ -0,0 +1,44 @@
+/*
+Write a program that creates a string that represents an 8×8 grid, using 
+newline characters to separate lines. At each position of the grid there is either 
+a space or a "#" character. The characters should form a chessboard.
+
+Passing this string to console.log should show something like this:
+
+ # # # #
+# # # # 
+ # # # #
+# # # # 
+ # # # #
+# # # # 
+ # # # #
+# # # #
+When you have a program that generates this pattern, define a binding size = 8 
+and change the program so that it works for any size, outputting a grid of the given width and height.
+*/
+let size = 8;
+for(let i = 0; i < size; i++)
+{
+  let stringForChess = "";
+  for(let k = 0; k < size; k++)
+  {
+    if(i % 2 === 0)
+    {
+      if(k % 2 === 0)
+      {
+      	stringForChess = stringForChess + " ";
+      }
+      else {stringForChess = stringForChess + "#";}
+    }
+    else
+    {
+      if(k % 2 === 0)
+      {
+      	stringForChess = stringForChess + "#";
+      }
+      else {stringForChess = stringForChess + " ";}
+    }
+    
+  }
+  console.log(stringForChess);
+}
```

Comment:
Probably 'program'  means 'function'. So please create function (choose name by yourself) and move your code in it.
Then you can call created function and pass any argument to it as 'size'.